### ISSUE TITLE
[AllyAudit] Disabled link will be disabled and conveyed to assistive tech [II]

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#3909]
**User Story**
As Spark we want Sparks Disabled Link to be conveyed to assistive technology. 

**Notes:**
Paciello Audit on Spark Design System
QLC-16 Inaccessible disabled link

**ISSUE DESCRIPTION**
The disabled link currently still receives focus, despite being disabled. The fact that the link is disabled is also not programmatically conveyed to assistive technologies.

While nominally disabled controls (including links) are exempt from color contrast requirements, it is also worth noting that the link does have a very low color contrast ratio of 1.55 (foreground color: #cfcfcf, background color: #ffffff).

**RECOMMENDATIONS**
In general, disabled controls should not receive focus. The simplest way to achieve this is not to use an actual live link with an href="..." attribute, but rather simply using an placeholder anchor element without it.

Alternatively, focus should be suppressed on the link using tabindex="-1", and the fact that the link is disabled should be conveyed programmatically using aria-disabled="true".

**AC**
This is for all frameworks: 
- Disabled link will not receive focus. because its disabled.
- the link is disabled should be conveyed programmatically using aria-disabled="true".
- Verify Color Contrast is accessible with Olya

**Comment by ssabo1** [comment]
Confirmed: Added to team backlog
